src/osd/scheduler/OpSchedulerItem.cc

```diff
@@ -12,8 +12,12 @@
  *
  */
 
-#include "OpQueueItem.h"
-#include "OSD.h"
+#include "osd/scheduler/OpSchedulerItem.h"
+#include "osd/OSD.h"
+
+namespace ceph {
+namespace osd {
+namespace scheduler {
```

Comment:
Since you combined the `namespace` declarations in other places (e.g., OpSchedulerItem.h) you could do so as well here.